blogpost1.php

```diff
@@ -0,0 +1,90 @@
+<html>
+  <head>
+    <title> Contact </title>
+    <link rel="stylesheet" href="style.css?Time=<?php echo microtime()?>" />
+  </head>
+  <body>
+      <h1>
+          More Info About Me </h1>
+          <div class = "active">
+                <ul>
+                <li>
+                    <a href = "index.php">
+                    Home
+                </a> </li>
+                    <li>
+                        <a href = "blogpost1.php">
+                            Contact
+                    </a> </li>
+
+                    <li>
+                        <a href = "Resume.php">
+                         Projects/Resume </a> </li>
+                  <li>
+                      <a href = "blogpost2.php">
+                          Summer Blog </a> </li>
+                </ul>
+          </div>
+
+     <br>
+     <div style='max-width: 90%; background-color: #B3C3E4; padding:10px; margin:auto; border-radius:10px'>
+<h2> Contact
+    <h4 class="contact">
+        Email: <a href="mailto:jenny.kreinik@lessannoyingcrm.com">jenny.kreinik@lessannoyingcrm.com <br/> </a>
```

Comment:
This is minor, but I'd put the `<br/>` outside of the `<a></a>` tag. Right now the line break is still surrounded by a link, so it seems possible a browser could interpret that as meaning that the start of the next line should still be clickable.